### ISSUE TITLE
Alimentar repositório com links criados em outro repositório

### ISSUE THREAD

**Issue by gabrielbdornas** [dados-mg/dados-mg.github.io#2]
@fjuniorr e @Andrelamor acredito que poderíamos começar a "rechear" este repositório. Como base, acredito que poderíamos utilizar as informações consolidadas [neste outro repositório](https://github.com/dados-mg/apresentacoes/blob/master/guias-materiais-dados-abertos.md). Neste sendido, nossa [página de documentação](https://docs.dados.mg.gov.br/) ficaria disponível com a "consolidação" dos materiais necessários para o treinamento sobre abertura de dados. Sugiro acrescentar:

- Documentação de setup de máquina;
- Vídeo treinamento abertura de dados Fhemig - "mão na massa" (necessário realizar upload para alguma ferramenta, sugiro YouTube).

**Comment by fjuniorr** [comment]
Além de começar o esforço de consolidação do material que estamos produzindo, esse final de ano também vai ser o momento de fazer um planejamento do conteúdo que precisamos produzir/refinar/coletar ao longo do ano que vem.

Apesar de ter gostado de cada artefato individual (ie. uma gravação, um deck de slides), ainda sinto falta de um encadeamento lógico entre eles.

Em especial, para servir como guia para os órgãos e entidades na "jornada" de abertura de dados, nossa documentação consolidada no https://docs.dados.mg.gov.br/, ainda que em múltiplas mídias, precisa ter uma sequência de início, meio e fim. 

Nossos dois últimos encontros me deixaram com a sensação de que ainda não temos uma resposta muito boa para "Por onde eu começo?" nem para "Qual o próximo passo?".

**Comment by gabrielbdornas** [comment]
@fjuniorr concordo totalmente, mas eu acho que o importante, neste momento, é começar. Por mais confuso que possa parecer [esta organização](https://github.com/dados-mg/apresentacoes/blob/master/guias-materiais-dados-abertos.md) lógica inicial criada é melhor do que não ter nada, ou ter que sempre ficar juntando alguns artefatos específicos para enviar para quem pedir. Com o tempo vamos substituindo esses links por materiais "nossos" mais bem elaborados e ou organizados. Mas enfim, não acho que tem uma resposta certa ou errada para esse assunto! O importante é que de um jeito ou de outro o pontapé inicial já foi dado com a apresentação de ontem (18/11/2021) para a Fhemig.
